test/suites/dev/moonbase/test-staking/test-staking-locks7.ts

```diff
@@ -1,11 +1,15 @@
 import "@moonbeam-network/api-augment";
 import { beforeAll, describeSuite, expect } from "@moonwall/cli";
 import { GLMR, MIN_GLMR_DELEGATOR, alith, generateKeyringPair } from "@moonwall/util";
-import { jumpRounds } from "../../../../helpers";
+import {
+  jumpRounds,
+  getDelegatorStakingFreeze,
+  getNumberOfDelegatorFreezes,
+} from "../../../../helpers";
 
 describeSuite({
-  id: "D023482",
-  title: "Staking - Locks - execute revoke",
+  id: "D013480",
```

Comment:
This change was also caused by a conflict, we should revert it to keep the test id order.